test/test.js

```diff
@@ -50,17 +49,22 @@ test('Storage', async t => {
       XDG_STATE_HOME
     }
   })
-  await timers.setTimeout(1000)
+  while (true) {
```

Comment:
Have you seen this test fail? In other words, what happens if the file `saturn-L2-node.log` is never created? (You can try this failure mode by temporarily changing the filename in the test to an invalid name.)

Is my assumption correct that this never-ending loop will stop when the test runner aborts the test on a timeout?

Reply:
Yes, it failed in the actions run linked above. The never ending loop will abort once the file exists